docs/.storybook/preview-head.html

```diff
@@ -1,11 +1,20 @@
-<link href='https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i' rel='stylesheet' type='text/css'></link>
+<link rel="preconnect" href="https://fonts.googleapis.com"></link>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin></link>
+<link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,400&display=swap" rel="stylesheet"></link>
```

Comment:
Using the new font v2 API.

Notice that we now load the following styles: `300, 400, 500, 600, 700` to cover both new and old requirements.